src/plugins/presentation_panel/public/panel_component/presentation_panel_internal.tsx

```diff
@@ -69,8 +69,8 @@ export const PresentationPanelInternal = <
   );
   const viewMode = rawViewMode ?? 'view';
 
-  const [initialLoadComplete, setInitialLoadComplete] = useState(!dataLoading);
-  if (!initialLoadComplete && (dataLoading === false || (api && !api.dataLoading))) {
+  const [initialLoadComplete, setInitialLoadComplete] = useState(false);
```

Comment:
This change is not going to work for maps. Maps can not start loading data until the the initial map is rendered. How about removing this change and we can discuss in a separate effort.